packages/replica-healthcheck/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as dotenv from 'dotenv'
```

Comment:
`src/index` usually exports the package as a library and executables usually go in something like `bin/health-check-server` or `exec/health-check-server`